packages/app-builder/src/models/get-operator-name.ts

```diff
@@ -7,6 +7,7 @@ import { isOperatorOption } from './operator-options';
 export function getOperatorName(
   t: TFunction<['common', 'scenarios'], undefined>,
   operatorName: string,
+  isFilter?: boolean,
```

Comment:
```suggestion
  isAggregationFilter?: boolean,
```
would be more understandable if you don't have the context I think